deploy/crds/integreatly.org_rhmis_crd.yaml

```diff
@@ -76,26 +66,22 @@ spec:
               - name
               - namespace
               type: object
+            rebalancePods:
+              type: boolean
             routingSubdomain:
               type: string
             selfSignedCerts:
               type: boolean
             smtpSecret:
-              description: "SMTPSecret is the name of a secret in the installation
-                namespace containing SMTP connection details. The secret must contain
-                the following fields: \n host port tls username password"
+              description: "SMTPSecret is the name of a secret in the installation namespace containing SMTP connection details. The secret must contain the following fields: \n host port tls username password"
               type: string
             type:
-              description: 'INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-                Important: Run "operator-sdk generate k8s" to regenerate code after
-                modifying this file Add custom validation using kubebuilder tags:
-                https://book.kubebuilder.io/beyond_basics/generating_crd.html'
+              description: 'INSERT ADDITIONAL SPEC FIELDS - desired state of cluster Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html'
               type: string
             useClusterStorage:
               type: string
-            rebalancePods:
-              type: boolean
           required:
+          - alertFromAddress
```

Comment:
I don't think this field is intended to be mandatory.
This is currently causing failure in e2e test in CI:
```
"error":"Could not create rhmi CR in redhat-rhmi-operator namespace: RHMI.integreatly.org \"rhmi\" is invalid: spec.alertFromAddress: Required value"
```